src/data/trainer-config.ts

```diff
@@ -640,6 +672,36 @@ export class TrainerConfig {
     return this;
   }
 
+  /**
+   * Sets random pokemon from the trainers team to instant tera. Uses their specialty types is they have one.
+   * @param count The amount of pokemon to have instant tera
+   * @returns this
```

Comment:
NIT: Match existing docs:
```suggestion
   * @returns this {@linkcode TrainerConfig}
```